components/todo-input.js

```diff
@@ -1,20 +1,72 @@
-import React from "react";
+import React, { useState } from "react";
 import styled from "styled-components";
 
-export default function TodoInput() {
-  return (
-    <Wrapper>
-      리액트-투두
-      <br />
-      시간을 입력하는 칸
-      <br />
-      Todo를 입력하는 칸
-    </Wrapper>
-  );
+export default function TodoInput({ addTodo }) {
+    const [thisTodo, setThisTodo] = useState({
+    })
+
+    const inputChangeHandler = (el) => {
+        const { name, value } = el.target;
+
+        if (name === 'dueDate') {
+            if (value.length > 8) {
+                el.target.value = value.slice(0, 8);
+                return;
+            }
+        }
+
+        setThisTodo({
+            ...thisTodo,
+            [name]: value
+        })
```

Comment:
```suggestion
        setThisTodo({
            ...thisTodo,
            [name]: name === 'dueDate' ? value.slice(0, 8) : value
        })
```
이렇게 리팩토링할 수 있습니다!
slice(0, 8)는 현재 value의 length가 8 미만이면 그냥 안 자르고 그대로 반환하기 때문에 추가로 조건문을 작성할 필요가 없어용